doc/source/contributing_code.rst

```diff
@@ -85,8 +85,10 @@ touching existing environments that may have stable versions of MDAnalysis. :
 
     .. code-block:: bash
 
-        conda create --name mdanalysis-dev
+        conda create --name mdanalysis-dev python==3.10
```

Comment:
why pin to 3.10?

Reply:
Because the only other place that a Python version is mentioned is somewhere far down (">= 3.9") and when you get to that part, it's too late to change your environment.

I wanted to add a sentence to the docs along "Look here to find out the minimum Python version currently supported by MDA and then use a Python version greater or equal to it" but I actually didn't really know what a good place to link to was. Writing "look in pyproject.yaml" seemed very off-putting, especially as we send beginners here.

If you have an idea then let me know.

I admittedly decided that I didn't have enough time to rewrite everything here (I ought to be finishing a paper or sleep).

Reply:
Could we just do >3.9 here?

Reply:
sure, I didn't think about using a `>`.